tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt

```diff
@@ -1,8 +1,8 @@
-# Tag: nvcr.io/nvidia/tensorrt:19.02-py3
-# Label: com.nvidia.cuda.version: 10.0.130
-# Label: com.nvidia.cudnn.version: 7.4.2
+# Tag: nvcr.io/nvidia/tensorrt:19.06-py3
+# Label: com.nvidia.cuda.version: 10.1.168
```

Comment:
so are we using different cuda/cudnn versions for windows vs linux?

Reply:
Linux CI machine is using docker container, where the cuda/cudnn version is chosen by Nvidia. Ideally windows machine should also update cuda/cudnn to their new versions if there is no side effect.